### DISCUSSION TITLE
Expansion Bay

### DISCUSSION THREAD

**Discussion by giscus** [framewiki/framewiki.net#17] [Talk]
# Expansion Bay

The unofficial community-driven Framework wiki

http://framewiki.net/expansion-bay

**Comment by Morpheus636** [comment]
More details about the technical details of the connector would be useful, e.g. the specific types of connectors (FXBeam, I think theres another kind of connector too). Also details about the FXBeam connector's durability (The public datasheets for that connector say its only rated for 50 cycles, but [Nirav said](https://community.frame.work/t/expansion-bay-connector-durability/31504/30?u=morpheus636) they're using a semi-custom part that's rated for significantly more).

**Comment by Morpheus636** [comment]
Also a section on hot-swap would be a good idea, since people thinking they can hot-swap the expansion bay is a common misconception.
